### ISSUE TITLE
Duplicate info with Astroconda doc

### ISSUE THREAD

**Issue by pllim** [SaraOgaz/STScI-env#11]
http://bashconda.readthedocs.io/en/latest/installing_anaconda.html has some duplicate info that is already explained in http://astroconda.readthedocs.io/en/latest/ . Do we need the info repeated here? Will that help or confuse people?

**Comment by SaOgaz** [comment]
It's funny you bring this up.  I did not originally have some of that information in there, but Warren recommended we put it into the doc so it would be less confusing for people, so there weren't any steps in this doc missing.  I generally agree with him.  I think people would be more upset with having to go to this "other" page just to get through the basic install, since we've got pretty much all the other necessary steps in here. On the other hand, it's a bit painful to be purposely repeating information...
